src/components/Dashboard/ProprietorPartnersListSm.vue

```diff
@@ -20,46 +20,55 @@
 
```

Comment:
What about the scenario where there are no parties?

Reply:
I added the condition for no parties and included unit test in my latest commit.

Reply:
Shouldn't it say "Not Entered" in this case?

Also, you don't need the "v-else" to protect the element at line 12: the v-for handles empty arrays just fine.